packages/integration-tests/src/entities/UserCodegen.ts

```diff
@@ -163,92 +166,94 @@ export abstract class UserCodegen extends BaseEntity<EntityManager> {
     "likedByUsers",
     "comment_id",
   );
-
   constructor(em: EntityManager, opts: UserOpts) {
     super(em, userMeta, UserCodegen.defaultValues, opts);
-    setOpts(this as any as User, opts, { calledFromConstructor: true });
+    setOpts((this as any) as User, opts, { calledFromConstructor: true });
   }
-
   get id(): UserId | undefined {
     return this.idTagged;
   }
-
   get idOrFail(): UserId {
     return this.id || fail("User has no id yet");
   }
-
   get idTagged(): UserId | undefined {
     return this.__orm.data["id"];
   }
-
   get idTaggedOrFail(): UserId {
     return this.idTagged || fail("User has no id tagged yet");
   }
-
   get name(): string {
     return this.__orm.data["name"];
   }
-
   set name(name: string) {
     setField(this, "name", cleanStringValue(name));
   }
-
+  /**
+   * Case is preserved. If searching use an ILIKE.
+   */
   get email(): string {
     return this.__orm.data["email"];
   }
-
+  /**
+   * Case is preserved. If searching use an ILIKE.
+   */
   set email(email: string) {
     setField(this, "email", cleanStringValue(email));
   }
-
+  /**
+   * This is an example of a multi
+   * line comment with
+   *
+   * line breaks and also to show that we just use headings, and not the level, allowing for more customisation in users docs.
+   */
   get ipAddress(): IpAddress | undefined {
     return this.__orm.data["ipAddress"];
   }
-
+  /**
+   * This is an example of a multi
+   * line comment with
+   *
+   * line breaks and also to show that we just use headings, and not the level, allowing for more customisation in users docs.
+   */
   set ipAddress(ipAddress: IpAddress | undefined) {
     setField(this, "ipAddress", ipAddress);
   }
-
+  /**
+   * Hashed with ultra secure base64.
+   */
   get password(): PasswordValue | undefined {
     return this.__orm.data["password"];
   }
-
+  /**
+   * Hashed with ultra secure base64.
+   */
```

Comment:
Wdyt of detecting when the comment has no newlines, and producing a one-liner comment like:

```
/** Hashed with ultra secure base64. */
```

Also perfectly fine to be a follow up/maybe someday.